site/en/docs/extensions/mv3/external_extensions/index.md

```diff
@@ -178,102 +148,156 @@ files. To see if this is the problem, follow these steps:
+For example:
+
+```json
+{
+  "external_update_url": "https://clients2.google.com/service/update2/crx",
+  "supported_locales": [ "en", "fr", "de" ]
+}
+```
+
 ## Using the Windows registry {: #registry }
 
-1.  Find or create the following key in the registry:
-    - 32-bit Windows: `HKEY_LOCAL_MACHINE\Software\Google\Chrome\Extensions`
-    - 64-bit Windows: `HKEY_LOCAL_MACHINE\Software\Wow6432Node\Google\Chrome\Extensions`
-2.  Create a new key (folder) under the **Extensions** key with the same name as the ID of your
-    extension (for example, `aaaaaaaaaabbbbbbbbbbcccccccccc`).
-3.  In your extension key, create a property, "update_url", and set it to the value:
-    "https://clients2.google.com/service/update2/crx" (this points to your extension's crx in the
-    Chrome Web Store):
+1. Find or create the following key in the registry:
+
+    32-bit Windows
+
+    : `HKEY_LOCAL_MACHINE\Software\Google\Chrome\Extensions`
+
+    64-bit Windows
 
+    : `HKEY_LOCAL_MACHINE\Software\Wow6432Node\Google\Chrome\Extensions`
+
+2. Create a new key (folder) under the **Extensions** key with the same name as the ID of your
+   extension. For example: `aaabbbcccdddeeefff`.
+3. In your extension key, create an "update_url" property and set it to the following value:
     ```json
     {
       "update_url": "https://clients2.google.com/service/update2/crx"
     }
     ```
+4. Launch Chrome.
+5. Go to **chrome://extensions**; you should see the extension listed.
+
+{% Aside %}
 
-4.  Launch the browser and go to **chrome://extensions**; you should see the extension listed.
+Check the [Windows registry common mistakes][section-faq-registry] for additional help.
+
+{% endAside %}
 
 ## Updating and uninstalling {: #updating }
 
 Google Chrome scans the metadata entries in the preferences and registry each time the browser
-starts, and makes any necessary changes to the installed external extensions.
+starts, and makes any necessary changes to the installed external extensions hosted in the Chrome
+Web Store.
 
-To update your extension to a new version, update the file, and then update the version in the
-preferences or registry.
+To update a local CRX file extension to a new version, update the file, and then update the version
+in the preferences json file.
 
 To uninstall your extension (for example, if your software is uninstalled), remove your preference
-file (aaaaaaaaaabbbbbbbbbbcccccccccc.json) or the metadata from the registry.
+file (for example, `aaabbbcccdddeeefff.json`) or the metadata from the registry.
 
 ## FAQ {: #faq }
 
 This section answers common questions about external extensions.
 
-**Will the methodology for allowing a "pre-install" still be supported by Google Chrome from M33
-onwards?**
-
-Yes, but only as an install from a Chrome Web Store `update_URL`, not from a local file path.
+### Will the methodology for allowing a "pre-install" still be supported by Google Chrome from M33 onwards?
```

Comment:
@rdcronin, any objection to removing this? M33 was released ~6.5 years ago. More broadly, I'm thinking we may want to adopt a default stance of removing content from > 5 years ago.

Reply:
I'm good to remove M33-specific content.  In this particular case, it seems like there might be value in keeping the question, but removing the M33:

"Is pre-install supported by Google Chrome?"

That said, I also don't quite understand the answer here - we allow off-store pre-install for enterprise distributions; are those out of scope here?

Reply:
> we allow off-store pre-install for enterprise distributions; are those out of scope here?

No, we've included off-store extensions in this document. We'll need to cleanup the answer.